### ISSUE TITLE
Closing parenthesis on new line and comment

### ISSUE THREAD

**Issue by axel-angel** [weavejester/cljfmt#72]
The following code has is weirdly indented after cljfmt:

```clojure
(defn evil
  []
  (println "evil!")
)

(defn function
  []
  ; FIXME: change name
  "hello")
```

becomes:

```clojure
(defn evil
  []
  (println "evil!")) (defn function
                       []
  ; FIXME: change name
                       "hello")
```

(why the comment isn't even indented to the code?)
but I would say the logical indentation should be:

```clojure
(defn evil
  []
  (println "evil!"))

(defn function
  []
  ; FIXME: change name
  "hello")
```

**Comment by weavejester** [comment]
Oh, interesting. Thanks for the report. I'll look into it.

**Comment by arrdem** [comment]
I can confirm finding instances of this in the cljfmt'd Jaunt codebase after the fact and not thinking of it / reporting it. Problems of having a large codebase and automatical tools.

**Comment by weavejester** [comment]
Should be fixed in 0.5.3.

**Comment by axel-angel** [comment]
Glad to hear, thanks.
